### ISSUE TITLE
[bug] ESLint doesn't detect indentation correctly with switch statements.

### ISSUE THREAD

**Issue by trusktr** [eslint/eslint#6490]
**What version of ESLint are you using?**
2.13.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**
```js
module.exports = {
    extends: 'eslint:recommended',
    parser: 'babel-eslint',
    parserOptions: {
        ecmaFeatures: {
            experimentalObjectRestSpread: true,
            jsx: true,
            modules: true, // import/export statements
        },
        sourceType: 'module',
    },
    env: {
        node: true,
        browser: true,
        commonjs: true,
        es6: true,
    },
    plugins: [
        'react'
    ],
    rules: {
        indent: ['error', 2],
        quotes: ['off', 'single'],
        'linebreak-style': [ 'error', 'unix' ],
        semi: ['off', 'always'],
        'no-undef': 'off',
        'no-console': 'off',
        'comma-dangle': 'off',
        'no-unused-vars': 'warn',
        'no-constant-condition': 'off',
    },
}
```

**What did you do? Please include the actual source code causing the issue.**
```js
Template.humanReadableAuditLog.helpers({
  results: resultsHelper,

  actionToWord(action) {
    var result = null

    switch(action) {
      case 'update':
        result = 'updated'
        break
      case 'insert':
        result = 'inserted'
        break
      case 'remove':
        result = 'removed'
        break
    }

    return result
  },
})
```

![screen shot 2016-06-20 at 7 41 40 pm](https://cloud.githubusercontent.com/assets/297678/16216363/0cbe43d6-371f-11e6-94f2-103951093961.png)

**What did you expect to happen?**
I expected that switch-statement indentation to be acceptable.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint doesn't think so.

I am able to fix the problem by indenting the code like this, with the `case` lines at the same level as the `switch`:

![screen shot 2016-06-20 at 7 43 59 pm](https://cloud.githubusercontent.com/assets/297678/16216411/5d40fdb2-371f-11e6-9379-bec9b4782a4c.png)

**Comment by trusktr** [comment]
Nevermind, I just learned of the `SwitchCase` option for the indent rule.
